DASH-IF-IOP.bs.md

```diff
@@ -10,8 +10,500 @@ Live discussion in #document-authoring on Slack.
+
+Clients MAY take advantage of any platform-specific optimizations for seamless playback that knowledge of period continuity enables; beyond that, clients SHALL treat continuity the same as connectivity.
+
+## Time shift window ## {#timing-timeshift}
+
+TSBD
+SPD
+
+Issue: Determine appropriate content for this section.
+
+## Real time clock synchronization ## {#timing-sync}
+
+Issue: Determine appropriate content for this section.
+
+## XLink ## {#timing-xlink}
+
+Issue: Determine appropriate content for this section.
+
+## Leap seconds ## {#timing-leapseconds}
+
+This section is intentionally left blank to indicate that the leap seconds topic is out of scope of this proposal.
+
+## Forbidden techniques ## {#timing-nonos}
+
+Some aspects of [[!MPEGDASH]] are not compatible with the interoperable timing model defined in this document. In the interest of clarity, they are explicitly listed here:
+
+* The `@presentationDuration` attribute SHALL NOT be used.
+
+## Dynamic MPD updates ## {#timing-mpd-updates}
+
+Dynamic MPDs MAY change over time. The nature of the change is not restricted unless such a restriction is explicitly defined.
+
+Some common reasons to make changes in dynamic [=MPDs=]:
+
+* Adding new [=media segments=] to an existing [=period=].
+* Adding new [=periods=].
+* Adjusting the durations and start times of [=periods=].
+* Removing [=media segments=] and/or [=periods=] that have fallen out of the time shift window.
+* Adding `MPD@mediaPresentationDuration` and (on the last [=period=]) `Period@duration` to signal that a live service has ended.
+* Converting the [=MPD=] to a static [=MPD=] to signal that a live service has become available on-demand as a recording.
+
+The following restrictions are defined here for MPD updates:
+
+* `MPD@id` SHALL NOT change.
+* `MPD@availabilityStartTime` SHALL NOT change.
+* `Period@id` SHALL NOT change.
+* `Period@start` SHALL NOT change.
+* The adaptation sets present in a [=period=] (i.e. the set of `AdaptationSet@id` values) SHALL NOT change.
+* The functional behavior of a representation (identified by a matching `Representation@id` value) shall not change, neither in terms of metadata-driven behavior (including metadata inherited from adaptation set level) nor in terms of segment timing.
+
+Issue: Do we need to restrict to "extend-only" behavior? https://github.com/Dash-Industry-Forum/DASH-IF-IOP/issues/224
+
+Issue: Review synchronization with 4.4.3.3
+
+Additional restrictions can be defined by other parts of this document.
+
+Clients SHOULD use `Period@id` to track [=period=] identity across [=MPD=] updates.
+
+The presence of `MPD@minimumUpdatePeriod` SHALL be used to signal that the [=MPD=] is expected to change. The value of this field indicates the frequency at which a client is expected to poll for [=MPD=] updates.
```

Comment:
Could be wrong, but IIRC setting `minimumUpdatePeriod=0` has a special meaning different for this one (I think it's related to in-band event signaling for manifest updates).

Reply:
I saw some statements about it but the more I read the more I got confused: https://github.com/Dash-Industry-Forum/DASH-IF-IOP/issues/218

Reply:
Comment from F2F: the value `0` is not a special case, all that text just tries to confusingly say what happens if this value is 0. All the normal rules still apply.